cmd/zetae2e/run.go

```diff
@@ -157,3 +163,45 @@ func parseCmdArgsToE2ETestRunConfig(args []string) ([]runner.E2ETestRunConfig, e
 	}
 	return tests, nil
 }
+
+// updateConfigWithDynamicERC20 loads ERC20 addresses via gRPC given CLI flags
+func updateConfigWithDynamicERC20(ctx context.Context, conf *config.Config, erc20ChainName, erc20Symbol string) error {
```

Comment:
I think it would be cleaner to have a function "findERC20(ctx, chainName, symbol) (asset, zrc20)"
And update the config in the calling function

Reply:
Updated